cmd/resources/serve.go

```diff
@@ -37,11 +39,20 @@ func (opts *serveOpts) setup(flags *pflag.FlagSet) {
 	flags.StringVar(&opts.Address, "address", "0.0.0.0", "Address to bind")
 	flags.IntVar(&opts.Port, "port", 8080, "Port on which the server will listen")
 	flags.StringArrayVarP(&opts.WatchPaths, "watch", "w", nil, "Paths to watch for changes")
+	flags.BoolVar(&opts.NoWatch, "no-watch", opts.NoWatch, "Do not watch for changes")
```

Comment:
I don't have a strong preference but this makes the UX a bit confusing – one could theoretically pass both `--watch` and `--no-watch` which is... Weird.

Do you think we should just get rid of `--watch`, and instead force the user to always watch the same dirs as they're evaluating?

Reply:
We need the `--watch` flag when `--script` is given though :zipper_mouth_face:

Reply:
Got it, let's stick to this approach then.